pyproject.toml

```diff
@@ -112,19 +88,9 @@ full = [
     "pandas==1.3.5; python_version == '3.7'",
     "pandas==2.0.3; python_version == '3.9'",
     "pandas==2.1.1; python_version > '3.9'",
-    "osmnx",
-    "pyvista==0.42.2; python_version > '3.7'",
-    "pyvista==0.38.0; python_version <= '3.7'",
-    "vtk==9.2.6",
-    "SRTM.py",
-    "utm",
-    "scikit-rf",
-    "openpyxl==3.1.2",
-]
-ci = [
-    "vtk-osmesa==9.2.20230527.dev0",
 ]
```

Comment:
We are using that specific vtk package to avoid having issues with opengl32.dll.
I expect the documentation to fail when running some examples.

Reply:
Nvm, everything is passing and it seems that our previous issues were related to the fact that we installed `pyvista` and/or `vtk`. Avoiding the use of `vtk-osmesa` is a good catch !